src/transformV2.ts

```diff
@@ -0,0 +1,157 @@
+import is from 'is';
+import Uri from 'jsuri';
+import marked from 'marked';
+
+import { BaseTemplateOptions, Colors, Rating, TemplateV2Options } from './base';
+import messages from './messages';
+
+export type TransformV2Options = {
+  choices?: string[];
```

Comment:
The definition combines unrelated fields from multiple question types. Can we have some nice interface, that will make clear what properties are related to question (interface per question type) and which fields are related to the email itself?